lib/orderbook/errors.ts

```diff
@@ -65,6 +66,10 @@ const errors = {
     message: `${currency} outbound balance of ${availableAmount} is not sufficient for order amount of ${amount}`,
     code: errorCodes.INSUFFICIENT_OUTBOUND_BALANCE,
   }),
+  EXCEEDING_LIMIT: (currency: string, amount: number, limit: number) => ({
```

Comment:
We should add this to the `INVALID_ARGUMENT` case in `getGrpcError` in GrpcService.ts.